pkgs/tools/cd-dvd/srt-to-vtt-cl/fix-which-command.patch

```diff
@@ -0,0 +1,8 @@
+--- a/Makefile
++++ b/Makefile
+@@ -1,4 +1,4 @@
+-ERR = $(shell which clang++ >/dev/null; echo $$?)
++ERR = $(shell @which@/bin/which clang++ >/dev/null; echo $$?)
```

Comment:
Is the code even necessary? The purpose of using `which`, as far as I can tell, is to generate an error message when `clang++` isn't found. If we substitute an absolute path to a `/nix/store/...` copy of `clang++`, that should be impossible.

Reply:
I see... So do you think I should remove this verification in this patch file?

Reply:
I'd do that in preference to adding dependencies to make the validation work, at least. If the derivation is doing its job right, it should be ensuring that whichever c++ compiler is needed is there 100% of the time (`stdenv` provides GCC's g++ out-of-the-box; if clang is needed, it should be an explicit dependency).